scripts/changelog.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+import os
+import argparse
+import datetime
+import subprocess
+from typing import List
+
+def ymd():
+    return datetime.datetime.now().strftime("%Y-%m-%d")
+
+def git_current_branch() -> str :
+    out = subprocess.check_output(["git", "branch", "--show-current"])
+    return out.strip().decode("utf-8")
+
+def get_dir() -> str:
+    here = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(
+        here,
+        "../.changelogs")
+
+def get_changefiles() -> List[str]:
+    changedir = get_dir()
+    changefiles = []
+    for f in os.listdir(changedir):
+        if f.endswith(".txt") and not f.startswith("."):
+            changefiles.append(os.path.join(changedir, f))
+
+    return changefiles
+
+def cmd_add(args):
+    text = args.text.strip()
+    if not text.startswith("- "):
+        text = "- %s" % text
+
+    timestamp = ymd()
+    branchname = git_current_branch()
+    fname = os.path.join(get_dir(), "%s-%s.txt" % (timestamp, branchname))
```

Comment:
```suggestion
    fname = os.path.join(get_dir(), "%s-%s.md" % (timestamp, branchname))
```
so that editors get the syntax right..

Reply:
Ah, good point!